src/main.ts

```diff
@@ -12,7 +12,5 @@ const UBIQUITY = `
 
 export default function main(app: Probot) {
   console.log(UBIQUITY);
-  // app.onAny(bindEvents as any);
-  app.on("issue_comment.created", bindEvents);
-  app.on("issue_comment.edited", bindEvents);
+  app.onAny(bindEvents as any);
```

Comment:
`as any` is code smell. You generally should strive to never use `as any` but instead, at least as a generic function definition or ideally a specific signature of a function.

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/0a2f5d727febbba6a726141ec04f0e1f83ecb6ab

Reply:
Agree with your opinion not to use `as any` but sometimes it was kinda difficult to define the function definitions assignable what the Probot wants. but yeah let me try it without using `as any`

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/7bf059daee4f16052e2d1f348e023e0428229622